content/posts/chapelcon25-retro/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Reflections on ChapelCon '25"
+date: 2025-11-05
+tags: ["ChapelCon", "Community"]
+series: []
+summary: "This post is a retrospective on ChapelCon '25"
+weight: 15
+authors: ["Brandon Neth"]
+---
+
+ChapelCon '25 is behind us, and another year of productive HPC programming lies ahead!
+Don't worry if you missed a talk, you can find the recordings and slides on the ChapelCon '25 [webpage](https://chapel-lang.org/chapelcon25/).
+Before reflecting on the conference, I'd like to thank everyone who made ChapelCon '25 the exciting week it was. 
+Thank you to contributors and participants for the exciting dialogue during our demo sessions and conference days.
+Thank you to the program committee for your support in reviewing a record-breaking number of submissions.
+And, of course, huge thank yous to Luca Ferranti for his work as Program Committee Chair, Daniel Fedorin for his as Tutorial Day Chair, and Jade Abraham for theirs as Office Hours Chair.
+
+### Second Conference, First Experiments
+
+As the second instance of ChapelCon, this year's conference was our first opportunity to experiment with some changes to the conference format.
+We tried out four changes to the conference: a season change from early summer to early autumn, an expansion from three days to four, a new format for tutorial days, and a new approach to the submission and review process.
```

Comment:
I'd suggest turning these four things into bullets, just to add a little visual variance to this article and break up the text.  It'd also make them a little easier to visually parse and iterate over.